composer.json

```diff
@@ -27,25 +27,25 @@
         "sonata-project/doctrine-extensions": "^1.1",
         "sonata-project/form-extensions": "^1.0",
         "sonata-project/twig-extensions": "^1.0",
-        "symfony/asset": "^4.3",
-        "symfony/config": "^4.3",
-        "symfony/console": "^4.3",
-        "symfony/dependency-injection": "^4.3",
+        "symfony/asset": "^4.3 || 5.0",
```

Comment:
Why the constraint allowing the branch 5 is not trusting semver? (`5.0` => `^5.0`).

Reply:
I realised just afterwards, it is fixed in https://github.com/sonata-project/SonataBlockBundle/pull/657